pleskdistup/common/src/mounts.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023-2024. WebPros International GmbH. All rights reserved.
+import os
+import typing
+
+
+def get_fstab_configuration_misorderings(configpath: str) -> typing.List[typing.Tuple[str, str]]:
+    """
+    Analyzes the fstab configuration file to find misorderings in mount points.
+    This function reads the fstab configuration file specified by `configpath` and checks for any misorderings
+    in the mount points. A misordering is defined as a mount point that appears before its parent directory
+    in the fstab file.
+    Args:
+        configpath (str): The path to the fstab configuration file.
+    Returns:
+        List[Tuple[str, str]]: A list of tuples where each tuple contains a misordered parent directory and
+        its corresponding mount point.
+    Example:
+        >>> get_fstab_configuration_misorderings('/etc/fstab')
+        [('/home', '/home/user'), ('/var', '/var/log')]
+    """
+
+    if not os.path.exists(configpath):
+        return []
+
+    mount_points_order = {}
+    with open(configpath, "r") as f:
+        for iter, line in enumerate(f.readlines()):
+            if line.startswith("#") or line == "\n" or line == "":
+                continue
+            mount_point = line.split()[1]
+            mount_points_order[mount_point] = iter
+
+    misorderings = []
+    for mount_point in mount_points_order.keys():
+        if mount_point == "/":
+            continue
+
+        if mount_points_order["/"] > mount_points_order[mount_point]:
```

Comment:
Is it impossible to have no `/` root entry?

Reply:
I expect that a `/` will usually be inside the file, however, it is not necessarily required as far as I know. So let's add a test and fix it.
Thanks for pointing it out.